code/sound/sound.cpp

```diff
@@ -1145,6 +1187,38 @@ int snd_is_playing(sound_handle sig)
 	return 0;
 }
 
+// ---------------------------------------------------------------------------------------
+// snd_is_paused()
+//
+// Determine if a sound is paused
+//
+// returns:			1				=> sound is currently paused
+//						0				=> sound is not paused
+//
+// parameters:		sig	=> signature of sound, what is returned from snd_play()
+//
+int snd_is_paused(sound_handle sig)
```

Comment:
So, since you're writing new functions, these should really just return bools.  Not that there's really any chance of this, but it's safer to return only either true or false and not accidentally something that the calling code is not equipped to handle.

Reply:
I can do that, it's really simple. It will be the only functions in the entire sound code that work that way, though.

Reply:
Done